src/main/java/racinggame/domain/CarPosition.java

```diff
@@ -0,0 +1,53 @@
+package racinggame.domain;
+
+public class CarPosition {
+
+    private static final int MIN_POSITION = 0;
+    private static final int MOVE_THRESHOLD = 4;
+    private final int position;
+
+    public CarPosition() {
+        this(MIN_POSITION);
+    }
+
+    public CarPosition(int position) {
+        this.position = position;
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public CarPosition move(int position) {
+        if (position >= MOVE_THRESHOLD) {
```

Comment:
```suggestion
    public CarPosition move(int randomValue) {
        if (randomValue >= MOVE_THRESHOLD) {
```

CarPosition에 move까지 메시지를 보내는 것은 생각해 보지 않았는데 이 구현도 나름 일리가 있음.